models/Student.js

```diff
@@ -23,13 +23,36 @@ const StudentSchema = new Schema({
     type: String,
     required: true
   },
+  phone_number: {
+    type: String,
+    required: true
+  },
+  dateOfBirth: {
+    type: String,
+    required: true
+  },
+  guardianEmail: {
+    type: String,
+    required: false
+  },
   email: {
     type: String,
     required: true
   },
+  guardianPhone: {
+    type: String,
+    required: true
+  },
+  notes: {
+    type: String,
+    required: true
+  },
+  interestsAndHobies: {
```

Comment:
- Hobbies/Interests do not save to the DB when **adding** a student. 

- Hobbies/Interests are not populating on the UI when I go to **edit** a student. They saved in the DB. But if you edit the same student again, the hobby information is deleted and the DB shows an empty string.

Here, you have `interestsAndHobies`. In db.js & edit-student.ejs, you have `interests`.
This name mismatch may be the root of some of the above issues.